vcr.yml

```diff
@@ -6,14 +6,18 @@ instance:
   region: aws.euw1
   entrypoint: [yarn, run-server]
   application-id: <APP_ID>
-  domains: [meet.vonagenetworks.net]
+  domains: <DOMAIN>
```

Comment:
yaml allows comments so we may want to add a note here that this symbol is here to be replaced by sed in  
`.github/workflows/deploy-to-vcr.yml`